pom.xml

```diff
@@ -138,6 +138,9 @@
   <packaging>pom</packaging>
 
   <modules>
+    <!-- sdks/java/build-tools has project-wide configuration. To make these available
+      in all modules, link it directly to the parent pom.xml. -->
+    <module>sdks/java/build-tools</module>
```

Comment:
Too many stuff at top-level makes it harder to navigate. Then, these are java-build tools, not so much Python or other SDKs.